binders/kafka-binder/spring-cloud-stream-binder-kafka-streams/src/main/java/org/springframework/cloud/stream/binder/kafka/streams/aot/KafkaStreamsBinderRuntimeHints.java

```diff
@@ -18,7 +18,6 @@
 
 import java.util.stream.Stream;
 
-import org.apache.kafka.streams.errors.DefaultProductionExceptionHandler;
 import org.apache.kafka.streams.kstream.GlobalKTable;
 import org.apache.kafka.streams.kstream.KStream;
 import org.apache.kafka.streams.kstream.KTable;
```

Comment:
How about other Apache Kafka types which will go away after Reachability Metadata contribution?

Reply:
Sorry,  missed this one. We are already using FQCNs for the others used from Kafka Streams. Is that what you meant?

Reply:
Never mind: this ore other way - you still would need to modify this class after Reachability Metadata contribution.
So, all good 😄 
Thank you for a great support!